modules/test/apis/server.go

```diff
@@ -18,11 +18,12 @@ import (
 	"net/http/httptest"
 
 	"github.com/go-logr/logr"
+	"github.com/gorilla/mux"
```

Comment:
can we make this in a separate commit? (can be in a same PR if you need it)

Reply:
done